jni/osd_dji_overlay_udp.c

```diff
@@ -126,6 +139,92 @@ static display_info_t *current_display_info;
 
 int event_fd;
 
+#define FAKEHD_ENABLE_KEY "fakehd_enable"
+static int fakehd_enabled = 0;
+static int fakehd_trigger_x = 99;
+static int fakehd_trigger_y = 99;
+
+static void check_is_fakehd_enabled()
+{
+    DEBUG_PRINT("checking for fakehd\n");
+    if (get_boolean_config_value(SPLASH_KEY))
```

Comment:
🤔 - I'll fix this when I rebase over the merged config branch

Reply:
Dammit. 

Took me about 10 minutes looking to spot what I'd done!